mathics/builtin/__init__.py

```diff
@@ -3,30 +3,108 @@
 
 
 from mathics.builtin import (
-    algebra, arithmetic, assignment, attributes, calculus, combinatorial, compilation,
-    comparison, control, datentime, diffeqns, evaluation, exptrig, functional,
-    graphics, graphics3d,
-    image, inout, integer, iohooks, linalg, lists, logic,
-    manipulate, quantities, numbertheory, numeric, options, patterns,
-    plot, physchemdata, randomnumbers, recurrence, specialfunctions, scoping,
-    strings, structure, system, tensors, xmlformat, optimization)
+    algebra,
```

Comment:
I wonder if we could DRY this using the list `modules` we define a few lines below...